fe/fe-core/src/main/java/org/apache/doris/qe/StmtExecutor.java

```diff
@@ -885,6 +891,18 @@ public void analyze(TQueryOptions tQueryOptions) throws UserException {
             if (parsedStmt instanceof QueryStmt) {
                 queryStmt = (QueryStmt) parsedStmt;
                 queryStmt.getTables(analyzer, false, tableMap, parentViewNameSet);
+
+                // queue query here
+                if (Config.enable_resource_group && Config.enable_query_queue) {
```

Comment:
why in analyze method? Should put it into queue after analysis and before send fragment to BE. So that if a query has 语法错误， then user could get analysis error as soon as possible.